src/main.cpp

```diff
@@ -1,6 +1,10 @@
 #include "tetris.h"
+#include <cstdlib>
+
 int main(int argc, char* argv[])
 {
-    auto game = Tetris(argc, argv);
```

Comment:
DONT use an assignment. This performs an implicit copy operation if the Tetris object! Instead...